definitions/divorce/json/CaseField/CaseField-nonprod.json

```diff
@@ -64,5 +64,61 @@
     "HintText": "The solicitor believes that the facts stated in this response are true",
     "FieldType": "YesOrNo",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "11/10/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "labelAmendPetitionForRefusalSol",
```

Comment:
You will need to add authorisation for each of the fields.
here you can use the -nonprod.json file (-prod.json not needed as the fields do not exist on prod).
Just add the: caseworker-divorce-solicitor role for each of those. With "CRU" permissions

Reply:
Seeing they are all labels, is the **C** required here or just **R**

Reply:
Done. Entries added for only **caseworker-divorce-solicitor** at this time